pages/_profiles/Person/0.3-DRAFT.html

```diff
@@ -0,0 +1,310 @@
+  property: curriculumVitae
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'A textual description of the professional experiences a person has
+    acquired. '
+  example: ''
+  expected_types:
+  - Text
+  marginality: Optional
+  property: experience
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'Textual description of current or past funding ths person has acquired '
+  example: ''
+  expected_types:
+  - Text
+  marginality: Optional
+  property: funding
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'Any interests held by the person. These should be single words or
+    short phrases that relate to a person '
+  example: ''
+  expected_types:
+  - Text
+  - URL
+  marginality: Optional
+  property: interest
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'Global connetions related to teaching and scholarly work in the past
+    3 years '
+  example: ''
+  expected_types:
+  - Text
+  marginality: Optional
+  property: internatonalActivities
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'Which journals this person is a reeree of '
+  example: ''
+  expected_types:
+  - CreativeWork
+  marginality: Optional
+  property: journalReferee
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: Keywords to describe the person. These should be single words or short
+    phrases that relate to a person. Keywords or tags used to describe this content.
+    Multiple entries in a keywords list are typically delimited by commas.
+  example: ''
+  expected_types:
+  - Text
+  marginality: Optional
+  property: keywords
+  type: bioschemas
+  type_url: ''
+- bsc_description: ''
+  cardinality: ONE
+  controlled_vocab: ''
+  description: A list of languages spoken by the Person Of a Person, and less typically
+    of an Organization, to indicate a known language. We do not distinguish skill
+    levels or reading/writing/speaking/signing here. Use language codes from the IETF
+    BCP 47 standard.
+  example: ''
+  expected_types:
+  - Language
+  marginality: Optional
+  property: knowsLanguage
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: 'All or notable publications, presentations, training materials, research
+    outputs or any other body of work attributed to this person. '
+  example: ''
+  expected_types:
+  - CreativeWork
+  marginality: Optional
+  property: works
+  type: bioschemas
+  type_url: ''
+name: Person
+parent_type: Person
+previous_release: ''
+previous_version: 0.2-DRAFT-2019_07_19
+redirect_from: '- "/devSpecs/Person/specification/"
```

Comment:
The redirection should exist only in the latest version so bioschemas.org/profiles/Person gets there. 
It should be removed from the previous version, in this case the file at ```pages/_profiles/Person/0.2-DRAFT-2019_07_19.html``` but that file was not modified so the redirection still gets to an old version rather that to the latest one. The GitHub actions should remove those lines from the previous version.